xml/libvirt_guest_installation.xml

```diff
@@ -302,15 +302,19 @@
         </step>
         <step>
           <para>
-            Under <guimenu>Enable boot menu</guimenu>, activate
-            <guimenu>Network PXE</guimenu> and confirm with
+            Under <guimenu>Enable boot menu</guimenu> and confirm with
             <guimenu>Apply</guimenu>.
+            You can also find the NIC device in the boot Menu configuration and move
```

Comment:
As a writer, I would like to add this information in a different way for ease of readability. Would you mind if I send a pull request to your branch?

Reply:
of course.